webapi/src/main/scala/org/knora/webapi/messages/v1/responder/valuemessages/ValueMessagesV1.scala

```diff
@@ -54,29 +58,54 @@ case class CreateValueApiRequestV1(project_id: IRI,
                                    res_id: IRI,
                                    prop: IRI,
                                    richtext_value: Option[CreateRichtextV1] = None,
+                                   link_value: Option[IRI] = None,
                                    int_value: Option[Int] = None,
                                    decimal_value: Option[BigDecimal] = None,
                                    boolean_value: Option[Boolean] = None,
                                    uri_value: Option[String] = None,
                                    date_value: Option[String] = None,
                                    color_value: Option[String] = None,
                                    geom_value: Option[String] = None,
-                                   link_value: Option[IRI] = None,
                                    hlist_value: Option[IRI] = None,
                                    interval_value: Option[Seq[BigDecimal]] = None,
                                    geoname_value: Option[String] = None,
-                                   comment: Option[String] = None)
+                                   comment: Option[String] = None) {
+
+    /**
+      * Returns the type of the given value.
+      *
+      * TODO: make sure that only one value is given.
```

Comment:
I think this should do it:

https://github.com/dhlab-basel/Knora/commit/c4c4ba07bd1e9c2a4253fd12cfd82f46861e55fa

Could you add a test for it?